app/main.py

```diff
@@ -1 +1,63 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if not isinstance(command, str):
+        print("Incorrect format")
+        return
+
+    cmd_list = command.strip().split(" ")
+
+    if len(cmd_list) and cmd_list[0] != "mv":
+        print("Command should be 'mv'")
+        return
+
+    if len(cmd_list) != 3:
```

Comment:
The error message on line 16 should be corrected to 'Command 'mv' should contain exactly two arguments' to fix the grammatical error.